content/en/entities/featuredtag.md

```diff
@@ -12,6 +12,7 @@ menu:
 {
   "id": "627",
   "name": "nowplaying",
+  "url": "https://foo.bar",
```

Comment:
If you have a better URL to suggest here (or maybe the field isn't mandatory and then the field description would need an update).